Sming/SmingCore/Network/rBootHttpUpdate.h

```diff
@@ -57,6 +57,10 @@ class rBootHttpUpdate: private HttpClient {
 	rboot_write_status rBootWriteStatus;
 	uint8 romSlot;
 	otaUpdateDelegate updateDelegate;
+
+    virtual void writeInit();
```

Comment:
Well, they are aligned in my editor - you are right, need to change my tab settings ;-)

Also changed the base class to protected to allow better http error handling within derived classes.

The git workflow is still very new to me and I really appreciate any feedback!